.github/workflows/test.yml

```diff
@@ -136,7 +136,6 @@ jobs:
           coverageFile: coverage.xml
           thresholdAll: 0.87
           thresholdNew: 0.9
-          thresholdModified: 0.9
```

Comment:
I figured this was something like "If a file was modified, 90% of the modified lines must still be covered by unit tests", but it's actually "If a file was modified, the whole file must now have 90% coverage." I don't think that's super helpful, so I removed it.